lib/rucio/daemons/hermes/hermes.py

```diff
@@ -365,10 +366,18 @@ def submit_to_elastic(messages: list[dict], endpoint: str, logger: "Callable") -
     :returns:                  HTTP status code. 200 and 204 OK. Rest is failure.
     """
     text = ""
+    elastic_username = config_get("hermes", "elasticsearch_username",
```

Comment:
one of the the configuration options used below is `elastic_endpoint`. Should we keep the same naming? `elastic_username` ?

Reply:
yes, honestly I thought I had named it that (and it what I have it called on my internal tests)! will make change now